iterative/testdata/script_template_cloud_gcp.golden

```diff
@@ -40,7 +40,7 @@ fi
 sudo npm config set user 0 && sudo npm install --global 18 value with "quotes" and spaces
 sudo tee /usr/bin/cml.sh << 'EOF'
 #!/bin/sh
-export GOOGLE_APPLICATION_CREDENTIALS_DATA='7 value with "quotes" and spaces'
+export GOOGLE_APPLICATION_CREDENTIALS_DATA=''
```

Comment:
Leaving blank because `7 value with "quotes" and spaces` is not a valid credentials JSON object and `CredentialsFromJSON` can't parse it.